canvas_modules/common-canvas/__tests__/common-properties/controls/passwordfield-test.js

```diff
@@ -175,6 +175,24 @@ describe("Passwordfield renders correctly", () => {
 		const messageWrapper = passwordWrapper.find("div.bx--form-requirement");
 		expect(messageWrapper).to.have.length(1);
 	});
+	it("Passwordfield eyeIcon tooltip appear correctly", () => {
```

Comment:
Can you add additional tests?
1. have the application override the labels as a single resource (test using harness)
2. use the defaults when the application didnt specify any

Reply:
You can take a look at this for example: https://github.com/elyra-ai/canvas/blob/main/canvas_modules/harness/src/intl/locales/en.json#L5
I added a custom label in the test harness to simulate an application specifying their own label override for the expression builder. Similarly, applications will specify their own overrides in their resources for the password icon tooltip